src/watts/plugin_abce.py

```diff
@@ -0,0 +1,84 @@
+# SPDX-FileCopyrightText: 2022 UChicago Argonne, LLC
+# SPDX-License-Identifier: MIT
+
+from datetime import datetime
+from pathlib import Path
+import os
+import sys
+import tempfile
+from typing import Mapping, List, Optional
+
+from .fileutils import PathLike
+from .parameters import Parameters
+from .plugin import TemplatePlugin
+from .plugin import Plugin
+from .results import Results
+
+
+class PluginABCE(TemplatePlugin):
+    """Plugin for running ABCE
+
+    Parameters
+    ----------
+    template_file
+        Templated ABCE input
+    extra_inputs
+        List of extra (non-templated) input files that are needed
+    extra_template_inputs
+        Extra templated input files
+    show_stdout
+        Whether to display output from stdout when ABCE is run
+    show_stderr
+        Whether to display output from stderr when ABCE is run
+
+    Attributes
+    ----------
+    executable
+        Path to ABCE executable
```

Comment:
This is a bit misleading to me. It appears to be the path to a directory, not an executable. Would it make more sense for the `executable` attribute to point directly to the `run.py` script?

Reply:
You are correct on both counts. I'll make that change.

Reply:
Looks like this comment still needs addressing -- down in the `__init__` method it's using the `ABCE_DIR` environment variable and then adding `run.py` onto that for the `execute_command` property. Either the description of `executable` here needs to change or what `executable` is actually pointing to.